src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -724,10 +726,71 @@ export class DBTCloudProjectIntegration
     );
   }
 
+  async getBulkCompiledSQL(models: NodeMetaData[]) {
+    const downloadArtifactsVersion = "0.37.20";
+    const currentVersion = this.getVersion()
+      .map((part) => new String(part))
+      .join(".");
+    if (semver.gte(currentVersion, downloadArtifactsVersion)) {
+      const compileQueryCommand = this.dbtCloudCommand(
+        new DBTCommand("Getting catalog...", [
+          "compile",
+          "--download-artifacts",
+          "--model",
+          '"' + models.map((item) => item.name).join(" ") + '"',
+          "--output",
+          "json",
+          "--log-format",
+          "json",
+        ]),
+      );
+      const { stderr } = await compileQueryCommand.execute(
```

Comment:
these downloaded artifacts will be stored locally?

Reply:
yes, they are located in target folder